examples/buckling/cylinder.cpp

```diff
@@ -117,7 +122,7 @@ int main(int argc, char* argv[])
   app.set_help_flag("--help");
   app.allow_extras()->parse(argc, argv);
 
-  nonlinear_options.force_monolithic = linear_options.preconditioner != Preconditioner::Petsc;
+  // nonlinear_options.force_monolithic = linear_options.preconditioner != Preconditioner::Petsc;
```

Comment:
@tupek2 should this line just be removed? i don't remember if i made this change or not